code_schemes/new_practices.json

```diff
@@ -1,71 +1,231 @@
+      "CodeID": "code-9f954561",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "use_of_manure",
+      "DisplayText": "use of manure",
+      "VisibleInCoda": true,
+      "NumericValue": 4,
+      "CodeID": "code-3885be1c",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "mulching",
+      "DisplayText": "mulching",
+      "VisibleInCoda": true,
+      "NumericValue": 5,
+      "CodeID": "code-e01e4db9",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "planting_cover_crops",
+      "DisplayText": "planting cover crops",
+      "VisibleInCoda": true,
+      "NumericValue": 6,
+      "CodeID": "code-1fd83981",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "terracing_gabions",
+      "DisplayText": "terracing/gabions",
+      "VisibleInCoda": true,
+      "NumericValue": 7,
+      "CodeID": "code-cc38d263",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "irrigation",
+      "DisplayText": "irrigation",
+      "VisibleInCoda": true,
+      "NumericValue": 8,
+      "CodeID": "code-e9a57ffe",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "soil_testing",
+      "DisplayText": "soil testing",
+      "VisibleInCoda": true,
+      "NumericValue": 9,
+      "CodeID": "code-07c669cd",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "organic_biological_pest_control_methods",
+      "DisplayText": "organic/biological pest control methods",
+      "VisibleInCoda": true,
+      "NumericValue": 10,
+      "CodeID": "code-b0ad8b73",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "avoid_using_in_organic_fertilizers_products",
+      "DisplayText": "avoid using in organic fertilizers/products",
+      "VisibleInCoda": true,
+      "NumericValue": 11,
+      "CodeID": "code-b872896a",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "use_of_lime",
+      "DisplayText": "use of  lime",
+      "VisibleInCoda": true,
+      "NumericValue": 12,
+      "CodeID": "code-92c6839f",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "fallowing",
+      "DisplayText": "fallowing",
+      "VisibleInCoda": true,
+      "NumericValue": 13,
+      "CodeID": "code-d07a7669",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "digging_dams",
+      "DisplayText": "digging dams",
+      "VisibleInCoda": true,
+      "NumericValue": 14,
+      "CodeID": "code-157fa77d",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "modern_technology",
+      "DisplayText": "modern technology",
+      "VisibleInCoda": true,
+      "NumericValue": 15,
+      "CodeID": "code-8ba006db",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "intercropping",
+      "DisplayText": "intercropping",
+      "VisibleInCoda": true,
+      "NumericValue": 16,
+      "CodeID": "code-0e85b8ce",
+      "CodeType": "Normal"
+    },
+    {
+      "StringValue": "use_of_fertilizers",
```

Comment:
which shouldn't be chosen if we could choose "use of organic fertilizers"?

Reply:
This covers those who din't specify the type of fertilisers they want to use. re- RDA